_data/simple-icons.json

```
@@ -4747,8 +4747,13 @@
 	},
 	{
 		"title": "EasyEDA",
-		"hex": "1765F6",
-		"source": "https://easyeda.com"
+		"hex": "5588FF",
```

I didn't see the color `#5588FF` on any icon. Can you point out which resource you used to pick the color?

In the meantime, I found a color icon from https://easyeda.com/page/legal header, and its color is `#1765F6`.

---

@xifan2333 I checked the header resource from the https://easyeda.com/page/download, its color is `#1765F6`.

Could you change it back to `#1765F6`?

```suggestion
		"hex": "1765F6",
```